### PR TITLE
Fix crash on 0 or 1 notifications

### PR DESCRIPTION
**Pull request by techtonik**
GitHub doesn't provide Link header in this case, see
https://github.com/sindresorhus/notifier-for-github-chrome/pull/69#issuecomment-202513803

### CODE REVIEW

File: extension/api.js [hunk 1]
```diff
@@ -116,7 +112,16 @@
...
 
 			if (data) {
-				cb(null, pages, interval);
+	                        if (linkheader === null) {
+					// 0 or 1 pages
+					cb(null, data.length, interval);
+  				} else {
+					const lastlink = linkheader.split(', ').find(link => {
+						return link.endsWith('rel="last"');
+					});
```
Comment (sindresorhus) [inline extension/api.js]:
```js
const lastlink = linkheader.split(', ').find(link => link.endsWith('rel="last"'));
```

File: extension/api.js [hunk 2]
```diff
@@ -116,7 +112,16 @@
 			}
 
 			if (data) {
-				cb(null, pages, interval);
+	                        if (linkheader === null) {
```
Comment (sindresorhus) [inline extension/api.js]: Wonky indentation

**Comment by techtonik** [comment]
I wish there was some real tests for those requests.

**Comment by gcochard** [comment]
Squash into 1 commit, then LGTM.

**Comment by techtonik** [comment]
Done.

**Comment by sindresorhus** [comment]
LGTM. @YuriSolovyov ?

**Comment by YuriSolovyov** [comment]
Code is looking good, I'll be able to test it this evening.

**Comment by YuriSolovyov** [comment]
> I wish there was some real tests for those requests.

That would be a lot easier, if app was structured better, maybe we need an issue to discuss that.

**Comment by YuriSolovyov** [comment]
Works.
Thanks @techtonik

**Comment by techtonik** [comment]
Thanks for review.

**Pull request merged**